### PR TITLE
Be compatible with Cro::HTTP 0.8

### PR DESCRIPTION
**Pull request by Altai-man**

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -2,6 +2,9 @@ Revision history for App-AizuOnlineJudge
 
 {{$NEXT}}
 
+0.0.3  2019-08-19T11:11:00+01:00
```
Comment (titsuki) [inline Changes]: `{{$NEXT}}` is the placeholder for `0.0.3`, so could you write as follows?:
```
{{$NEXT}}
    - Fix compatibility with cro 0.8
```

File: META6.json [hunk 2]
```diff
@@ -22,5 +22,5 @@
   "source-url" : "git://github.com/titsuki/p6-App-AizuOnlineJudge.git",
   "tags" : [ ],
   "test-depends" : [ ],
-  "version" : "0.0.2"
+  "version" : "0.0.3"
```
Comment (titsuki) [inline META6.json]: I always use mi6 as a authoring tool and `mi6 release` command automatically increase the version field to the `0.0.3` from `0.0.2`. So, could you set this field to `0.0.2` as before?

**Comment by titsuki** [comment]
@Altai-man Thanks for your PR! I commended to the PR.

File: Changes (hunk 1 above)
Comment (Altai-man) [reply Changes]: Ah, thank you, had no idea how the format works and tried to guess. :)

**Comment by titsuki** [comment]
@Altai-man Thanks!

**Pull request merged**